src/Settings.ts

```diff
@@ -13,6 +13,10 @@ export namespace Settings {
         return !!_getMavenSection<boolean>("maven.showInExplorerContextMenu");
     }
 
+    export function warnConflictsInDiagnostics(): boolean {
+        return !!_getMavenSection<boolean>("warnConflictsInDiagnostics");

```

Comment:
For a "resource" scope setting, you should get its value passing a "resource uri". Or to make it "window"-scoped